### PR TITLE
Bug Report + New Package templates

### PR DESCRIPTION
**Pull request by miabbott**
I took a guess at what would be good for the bug report template.  The new package template is taken from #655.

### CODE REVIEW

**Comment by miabbott** [comment]
@bgilbert I didn't realize that creating templates would generate a PR otherwise I would have just closed #655 and gone this route.  My bad!  :bow:

File: .github/ISSUE_TEMPLATE/requesting-new-package.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+name: Requesting New Package
+about: Request a new package be added to Fedora CoreOS
+title: 'New Package Request: <package name>'
+labels: ''
+assignees: ''
+
+---
+
+Please answer the following questions about the package you are requesting:
```
Comment (dustymabe) [inline .github/ISSUE_TEMPLATE/requesting-new-package.md:10]:
```suggestion
Please try to answer the following questions about the package you are requesting:
```

It's definitely a lot of questions. Sometimes some of them aren't relevant, etc. Let's lighten the language a bit.

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**System Details:**
+ - Bare Metal/QEMU/AWS/GCP/etc
+ - Fedora CoreOS version
+
+**Ignition Configuration**
+Please attach your Ignition configuration used to provision you system.
```
Comment (dustymabe) [inline .github/ISSUE_TEMPLATE/bug-report.md:27]:
```suggestion
Please attach your FCCT or Ignition configuration used to provision your system. If not using FCCT to generate your Ignition config, does the Ignition config pass validation using [ignition-validate](https://coreos.github.io/ignition/getting-started/#config-validation)?
```

**Review by dustymabe — APPROVED** [review]
LGTM. A few suggestions

**Comment by dustymabe** [comment]
If we add issue templates, do we need an "other" template that's just blank for something other than bug report of package add request? I don't want users to turn away because they don't see an option that matches what they're looking for.

**Comment by miabbott** [comment]
> If we add issue templates, do we need an "other" template that's just blank for something other than bug report of package add request? I don't want users to turn away because they don't see an option that matches what they're looking for.

Good point.  I took the easy route with these and used the GUI to generate this PR, but it looks like I should have [RTFM](https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/about-issue-and-pull-request-templates) and tried something a bit more elegant.

Anyways, let's configure this as is and I'll do a follow-up with a free-form template.

**Comment by dustymabe** [comment]
:+1: - maybe squash the commits into 1 and then merge?

**Comment by miabbott** [comment]
> +1 - maybe squash the commits into 1 and then merge?

LOL.  squashing isn't enabled for the repo...let me try to change that.

**Comment by bgilbert** [comment]
There's a link to open a blank issue at the bottom of the [chooser](https://github.com/coreos/fedora-coreos-tracker/issues/new/choose).

**Pull request merged**